packages/contentstack-utilities/src/auth-handler.ts

```diff
@@ -378,6 +384,18 @@ class AuthHandler {
     );
   }
 
+  async getAuthorisationType(): Promise<any> {
+    return configHandler.get(this.authorisationTypeKeyName) ? configHandler.get(this.authorisationTypeKeyName) : false;
+  }
+
+  async isAuthorisationTypeBasic(): Promise<boolean> {
+    return configHandler.get(this.authorisationTypeKeyName) === this.authorisationTypeAUTHValue ? true : false;
+  }
+
+  async isAuthorisationTypeOAuth(): Promise<boolean> {
+    return configHandler.get(this.authorisationTypeKeyName) === this.authorisationTypeOAUTHValue ? true : false;
+  }
```

Comment:
Suggestion: All the above functions try to access from config(Means accessing file system every function call). Can we have a local state manage those values and return in the function? Unless/until changes happened in config it does need not to access config. [MINOR]

Reply:
@antonyagustine , config library internally managing this state as per my understanding from the library. So i think we dont need this handling here.

Reply:
@antonyagustine The Conf package internally handles this.